internal/cli/project_apply.go

```diff
@@ -454,6 +467,21 @@ func (c *ProjectApplyCommand) Flags() *flag.Sets {
 			Default: "30s",
 			Usage:   "Interval between polling if polling is enabled.",
 		})
+
+		f.BoolVar(&flag.BoolVar{
+			Name:    "app-poll",
```

Comment:
Should this be `status-poll`? What else are you expecting to poll in the future at the app level?

Reply:
Nothing at the moment! I think I'll go with `app-status-poll` since it's a flag on project, I think that's a bit clearer.